HRIS.Services.Tests/Services/EmployeeServiceUnitTests.cs

```diff
@@ -37,8 +38,11 @@ public EmployeeServiceUnitTests()
         employeeTypeServiceMock = new Mock<IEmployeeTypeService>();
         employeeAddressServiceMock = new Mock<IEmployeeAddressService>();
         _errorLoggingServiceMock = new Mock<IErrorLoggingService>();
+        _emailService = new Mock<IEmailService>();
         roleServiceMock = new Mock<IRoleService>();
-        employeeService = new EmployeeService(employeeTypeServiceMock.Object, _dbMock.Object, employeeAddressServiceMock.Object, roleServiceMock.Object, _errorLoggingServiceMock.Object);
+        employeeService = new EmployeeService(employeeTypeServiceMock.Object, _dbMock.Object,
```

Comment:
Refactor: is this a local or member variable? May need clean this for better readability.

Reply:
Keep in mind that the `EmployeeService` constructor had to be changed and so `EmployeeServiceUnitTests` also had to be changed. EmployeeService will need to be refactored and with that the unit tests will be looked at.